modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -100,28 +100,49 @@ spec:
       defaultPlugins:
       - openshift
       - aws
-    restic:
+    nodeAgent: <1>
       enable: false
+      uploaderType: restic
   snapshotLocations:
     - velero:
         config:
-          credentialsFile: /tmp/credentials/openshift-adp/cloud-credentials-credentials <1>
-          enableSharedConfig: "true" <2>
-          profile: default <3>
-          region: ${REGION} <4>
+          credentialsFile: /tmp/credentials/openshift-adp/cloud-credentials-credentials <2>
+          enableSharedConfig: "true" <3>
+          profile: default <4>
+          region: ${REGION} <5>
         provider: aws
 EOF
 ----
 
+<1> If you are using OADP 1.2, replace this configuration:
 +
-<1> The `credentialsFile` is the mounted location of the bucket credential on the pod.
-<2> The `enableSharedConfig` allows the `snapshotLocations` to share or reuse the credential defined for the bucket.
-<3> Assume your Velero default for your `profile: default`.
-<4> Specify `region` as your AWS region. This must be the same as the cluster region.
+[source,terminal]
+----
+nodeAgent:
+  enable: false
+  uploaderType: restic
+----
++
+with the following:
++
+[source,terminal]
+----
+restic:
+  enable: false
+----
++
+<2> The `credentialsFile` is the mounted location of the bucket credential on the pod.
+<3> The `enableSharedConfig` allows the `snapshotLocations` to share or reuse the credential defined for the bucket.
```

Comment:
Use "object", "resource", "custom resource", "spec", etc. as appropriate after the object reference. This helps with clarity and readability. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

```suggestion
<2> The `credentialsFile` field is the mounted location of the bucket credential on the pod.
<3> The `enableSharedConfig` field allows the `snapshotLocations` to share or reuse the credential defined for the bucket.
```

Reply:
@abrennan89 
Done